doc/doc-style-rules.md

```diff
@@ -0,0 +1,134 @@
+# Documentation style rules
+
+Thank you for helping us improve the documentation! Below we have described some rules that we
+advise you to follow when you expand the documentation with new articles or add new sections to it.
+
+Any comments should be written in the following format:
+
+```
+> **Note**
+> Text of
+> note
+```
+
+The same goes for Warning:
+
+```
+> **Warning**
+> Text of
+> your warning
+```
```

Comment:
No.

Reply:
I've changed my mind, after looking at https://github.com/vlang/v/pull/17294 . I did not know that github renders such notes with a small (i) icon next to them - it does look distinct and it can improve readability indeed.